src/app/services/filterSrv.js

```diff
@@ -193,11 +193,13 @@ define([
     // Return fq string for constructing a query to send to Solr.
     // noTime param use only in ticker panel so the filter query will return without
     // time filter query
-    this.getSolrFq = function(noTime) {
+    this.getSolrFq = function(noTime, sep) {
```

Comment:
Yes, the separator is added here because graph streaming expressions does not use the `&` in URL query parameters, instead it is embedded inside the expression